src/views/Validation/Validation.js

```diff
@@ -36,11 +36,11 @@ class Validation extends Component {
     }
 
     componentDidMount() {
-        if (this.props.jobs.length === 0) {
-            this.props.onRequestLogList();
-            this.props.onRequestSplitList();
-            this.props.onRequestJobs();
-        }
+        // if (this.props.jobs.length === 0) {
+        //     this.props.onRequestLogList();
+        //     this.props.onRequestSplitList();
+        //     this.props.onRequestJobs();
+        // }
         ReactGA.initialize('UA-143444044-1');
```

Comment:
please add these few lines (ReactGA...) to all the new pages you created in order to allow us to monitor the traffic in the analytics platform